### ISSUE TITLE
Tooltip Tabstop Light Theme: Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#903]
Original Auditor Issue: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/4ebce85a-7c6a-11ec-a3c2-cf27c827159a?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=1&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

## Description
Light Theme: Text content lacks 4.5:1 contrast ratio

### Element Description:
-The text written inside the circle icons "1, 2, 3" are having the same color as the background due to which the user can not perceive them visually. 

Text color code: #FFFFFF
Background color code: #FFFFFF
Contrast ratio: 1:1

### RULE : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

### HOW TO FIX: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

### REFERENCE: 
Deque University: https://dequeuniversity.com/class/visual-design2/contrast/text-against-background

### BACKGROUND: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

![color contrast issue on tooltip tabstop in light mode](https://user-images.githubusercontent.com/1062039/219795548-9d107b04-043c-4b97-8b9b-ade695ccdeeb.png)

**Comment by scurker** [comment]
This is just a cauldron documentation issue, and shouldn't show up where it's used elsewhere.
